FF1Lib/Treasure.cs

```diff
@@ -14,7 +14,7 @@ public partial class FF1Rom : NesRom
 		public const int TreasureCount = 256;
 
 		public const int lut_MapObjTalkJumpTblAddress = 0x390D3;
-		public const string giveRewardRoutineAddress = "93DD";
+		public const string giveRewardRoutineAddress = "06B0";

```

Comment:
so if we do that we need to update almost all the talkroutines, which will be a pain, I'd suggest putting a jump at DD93 to B006 instead

Reply:
Should not be a problem, as neither DD93 nor 93DD exists in the randomizer files, with the above exception, that i will corect.

Reply:
DD93 most likely is compromised, as i inserted some instructions in the OpenTreasureChest routine